heuristic/repair_operators/random_repair.py

```diff
@@ -32,4 +32,6 @@ def random_repair(current: Solution, rnd_state: RandomState) -> Solution:
             route = create_single_customer_route(customer)
             current.routes.append(route)
 
+    for route in current.routes:
+        route.sort_start()
```

Comment:
Space after this - but do we want to keep this here? Maybe just reserve sorting stuff for local search instead?